ui/App/OrgIdentity.svelte

```diff
@@ -0,0 +1,38 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="typescript">
+  import { Avatar, CopyableIdentifier } from "ui/DesignSystem";
+  import type * as ensResolver from "ui/src/org/ensResolver";
+
+  export let orgAddress: string;
+  export let registration: ensResolver.Registration | undefined = undefined;
+  export let style: string | undefined = undefined;
```

Comment:
We always include `margin-right: 0.5rem` in the style to ensure an appropriate spacing between the avatar and the domain or org address. It should not be the responsibility of the caller to ensure the spacing and we should hard-code this here.

Except for this the style is only used to add a left margin to the `Avatar`. This can also be accomplished by adding margins to the previous element at the call site. This is preferable to adding the style here and would allow us to eliminate the `style` property altogether.

Reply:
I disagree with this point.

I think it's best to leave positioning and spacing up to the enclosing elements. By abiding this principle throughout the codebase it makes it easier to reason about how something will look without having to look at the component itself or having to preview a render. Knowing that components won't touch their outer positioning makes the mental model easier to deal with for me. I think there's also a benefit in terms of component composition.

So far we've been following this principle and I don't see a good reason to break it now.
To avoid always setting the margin from outside we might use some flexbox styling, e.g. `gap`.

Reply:
Never mind, this was actually an internal spacing 🤦.

Reply:
0ea5679cb704a4cce5f9e9c845b1888ea8377b21